src/Agent.Listener/Configuration/NativeWindowsServiceHelper.cs

```diff
@@ -324,15 +324,14 @@ public bool IsValidCredential(string domain, string userName, string logonPasswo
 
         public NTAccount GetDefaultServiceAccount()
         {
-            SecurityIdentifier sid = new SecurityIdentifier(WellKnownSidType.LocalServiceSid, domainSid: null);
+            SecurityIdentifier sid = new SecurityIdentifier(WellKnownSidType.NetworkServiceSid, domainSid: null);
```

Comment:
Is it OK to always use NetworkService as the default? I would think it is fine, but I don't know why the old agent preferred LocalService on a non-domain joined machine.